src/ASM/LR/ASMu3Dmx.h

```diff
@@ -66,10 +66,7 @@ class ASMu3Dmx : public ASMu3D, private ASMmxBase
   virtual ~ASMu3Dmx() {}
 
   //! \brief Returns the spline surface representing the basis of this patch.
```

Comment:
Correct the doxy also, here we are dealing with volumes

Reply:
Fixed.